heat/core/tests/test_statistics.py

```diff
@@ -176,6 +178,64 @@ def test_argmin(self):
         with self.assertRaises(ValueError):
             ht.argmin(data, axis=-4)
 
+    def test_cov(self):
+        x = ht.array([[0, 2], [1, 1], [2, 0]], dtype=ht.float, split=1).T
+        if x.comm.size < 3:
+            cov = ht.cov(x)
+            actual = ht.array([[1, -1], [-1, 1]], split=0)
+            self.assertTrue(ht.equal(cov, actual))
+
+        f = h5py.File('heat/datasets/data/iris.h5', 'r')
```

Comment:
This is a bad idea. h5py is an optional feature, it might not necessarily be installed. We should fall back here to a iris.csv version that we should also have in the datasets.

it might as well be useful for testing purposes to use ht.load_csv instead

Reply:
i found a bug in load_csv. i have made an issue (#368) at the moment this problem does not allow for the the usage of load_csv.

Reply:
Yes, I have assigned Simon already. So we need to wait here for a proper load_csv()